xunit.testlogger.sln

```diff
@@ -20,50 +25,79 @@ Global
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86
 	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{7D81A423-EA7B-44C1-8406-A35B82C6854D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{7D81A423-EA7B-44C1-8406-A35B82C6854D}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{7D81A423-EA7B-44C1-8406-A35B82C6854D}.Debug|x64.ActiveCfg = Debug|x64
```

Comment:
Interesting. These were added by the `dotnet sln` command :)

Reply:
I'm not worrying about this. Is that okay?

Reply:
No worries.